lib/kvs/kvs_cursor.c

```diff
@@ -1171,12 +1178,12 @@ kvs_cursor_val_copy(
         memcpy(buf, vt->vt_data, min_t(u64, vt->vt_xlen, bufsz));
```

Comment:
This direct use of vt_xlen looks suspicious...

Reply:
Maybe open a ticket to review this if gaurav doesn't know offhand..

Reply:
I would agree with you. Seems like it should be changed to use the API.